pkgs/development/python-modules/gpy/default.nix

```diff
@@ -55,6 +64,5 @@ buildPythonPackage rec {
     changelog = "https://github.com/SheffieldML/GPy/releases/tag/v.${version}";
```

Comment:
the path for the release logs has changed (it's now `.../v${version}` instead of `.../v.${version}`

Reply:
Fixed.